values.yaml

```diff
@@ -0,0 +1,175 @@
+global:
+  # Enable OpenShift specific settings
+  openshift: false
+
+# Settings for argus
+argus:
+  # Image for deployment
+  image:
+    repository: "releaseargus/argus"
+    tag: "0.11.1"
+    registry: 'docker.io'
+    pullPolicy: IfNotPresent
+
+  containerSecurityContext: {}
+
+  # Additional InitContainers
+  extraInitContainers: null
+
+  # Additional sidecar containers
+  extraContainers: null
+
+  # List of extra env vars
+  extraEnvironmentVars: []
+    # - name: ARGUS_UID # Optional UID override
+    #   value: "911"
+    # - name: ARGUS_GID # Optional GID override
+    #   value: "911"
+
+  # List of Secrets to use as env var
+  extraSecretEnvironmentVars: []
+    # - envName: AWS_SECRET_ACCESS_KEY
+    #   secretName: vault
+    #   secretKey: AWS_SECRET_ACCESS_KEY
+
+  extraDeploymentLabels: {}
+  deploymentAnnotations: {}
+
+  extraPodLabels: {}
+  podAnnotations: {}
+
+  containerPort: 8080
+
+  # Priotiry class
+  priorityClassName: ""
+
+  # Add volumeMounts to
+  # Argus Contianer
+  extraVolumeMounts: null
+  # - name: scripts
+  #   mountPath: /scripts
+  #   readOnly: true
+
+  # Add Volumes to
+  # Argus Pod
+  extraVolumes: null
+  # - name: scipts
+  #   emptyDir: {}
+
+  storage:
+    # Create and Mount pvc
+    # Mounted under /app/data
+    enabled: true
+
+    # Set name for data pvc
+    name: ""
+
+    # Size of PVC
+    size: 10Gi
```

Comment:
Minor thing, but I think 10Gi is too high of a default for `argus.db` as it's only a small SQLite DB. Maybe just `10Mi`/`100Mi`/`1Gi`
Everything else is looking good to me though. Thanks for doing all his :

Reply:
Have reduced the size to 1Gi